content/en/blog/baremetal_performance/index.md

```diff
@@ -0,0 +1,152 @@
+---
+title: "How to achieve baremetal performance inside your virtual machine"

```

Comment:
Could you shorten this to "Achieving baremetal performance within a Virtual Machine"

Reply:
I don't know if baremetal is one word, two or hyphenated. Would be good to check style guides (e.g. Google's)